supabase/README.md

```diff
@@ -0,0 +1,11 @@
+# Supabase
+
+To generate types dynamically from Supabase, run the following from command line (in location `/website/supabase/`):
+
+```bash
+npx supabase gen types --lang=typescript --project-id <project-id> --schema public > database.types.ts
+```
```

Comment:
Preliminary docs, this is the command for generating the types from the schemas/definition in the cloud project locally